payment_paypal/indico_payment_paypal/translations/de_DE/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,96 @@
+
+# German (Germany) translations for PROJECT.
+# Copyright (C) 2017 ORGANIZATION
+# This file is distributed under the same license as the PROJECT project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2017.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2017-06-26 09:51+0200\n"
+"PO-Revision-Date: 2017-06-26 09:56+0200\n"
+"Last-Translator: Dennis Riehle <webmaster@riehle-web.com>\n"
+"Language: de_DE\n"
+"Language-Team: German (Germany) (http://www.transifex.com/projects/p/indico/language/de_DE/)\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.3.4\n"
+
+#: indico_payment_paypal/controllers.py:117
+msgid "Your payment request has been processed."
+msgstr "Ihr Zahlungsvorgang wurde bearbeitet."
+
+#: indico_payment_paypal/controllers.py:125
+msgid "You cancelled the payment process."
+msgstr "Sie haben den Zahlungsvorgang abgebrochen."
+
+#: indico_payment_paypal/plugin.py:34
+msgid "API URL"
+msgstr "API URL"
+
+#: indico_payment_paypal/plugin.py:34
+msgid "URL of the PayPal HTTP API."
+msgstr "URL der PayPal HTTP API."
+
+#: indico_payment_paypal/plugin.py:35 indico_payment_paypal/plugin.py:41
+msgid "Business"
+msgstr "Business"
+
+#: indico_payment_paypal/plugin.py:36
+msgid ""
+"The default PayPal ID or email address associated with a PayPal account. "
+"Event managers will be able to override this."
+msgstr ""
+"Die PayPal-ID oder E-Mail-Adresse des PayPal-Accounts, welcher standardmäßig "
+"verknüpft werden soll. Dies kann pro Veranstaltung überschrieben werden."
```

Comment:
"benutzt" or "verwendet" instead of "verknüpft"?

Also, do we really translate "Event" (especially for events like meetings "Veranstaltung" doesn't seem the best choice) in other places (to be checked in the glossary)? Keeping the anglicism "Event" isn't so uncommon in German....